### PR TITLE
SI-5817: Add header to language.scala and languageFeature.scala

### PR DESCRIPTION
**Pull request by kanielc**
These two files were missing headers.
There is other work on-going to automate updating of the headers, but
that likely won't crossover with this.

### CODE REVIEW

File: src/library/scala/language.scala [hunk 1]
```diff
@@ -1,3 +1,13 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___     Scala API                            **
+**    / __/ __// _ | / /  / _ |    (c) 2003-2015, LAMP/EPFL             **
+**  __\ \/ /__/ __ |/ /__/ __ |    http://scala-lang.org/               **
+** /____/\___/_/ |_/____/_/ | |                                         **
+**                          |/                                          **
+\*                                                                      */
+
+
+
```
Comment (som-snytt) [inline src/library/scala/language.scala:10]: I guess the three blank lines are a signal to your alien conspirators that they should destroy the Earth in exactly three days.

That gives you plenty of time to repair your craft which was damaged on entry.

Comment (kanielc) [inline src/library/scala/language.scala:10]: They were the same blank lines in native.scala, from which I took this header.
Easy to remove if you so wish it.

**Comment by adriaanm** [comment]
Thanks, looks like I'll have to hold off until 2015 to merge to avoid the lawyers tracking us down over the holidays :-)

**Comment by retronym** [comment]
LGTM

**Pull request merged**